tldw_Server_API/app/core/DB_Management/backends/pg_rls_policies.py

```
@@ -215,49 +212,46 @@ def add(sql: str) -> None:
-    """
+def _ensure_rls_policy_set(
+    backend: DatabaseBackend,
+    *,
+    name: str,
+    statements: list[str],
+) -> bool:
     try:
-        if not hasattr(backend, 'backend_type') or backend.backend_type.name != 'POSTGRESQL':
+        if not hasattr(backend, "backend_type") or backend.backend_type.name != "POSTGRESQL":
             return False
     except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)